src/lib/BarStore.js

```diff
@@ -87,7 +87,11 @@ export default class BarStore {
 
   async fetchAppsList () {
     const apps = await this.fetchApps()
-    const comingSoonApps = await this.fetchComingSoonApps()
+    let comingSoonApps = await this.fetchComingSoonApps()
+    comingSoonApps = comingSoonApps
+      // drop coming soon apps already installed
+      .filter(comingSoonApp =>
+        apps.filter(app => app.slug === comingSoonApp.slug))
     this.appsList = apps.concat(comingSoonApps)
```

Comment:
@enguerran Actually this part of the code caused at least three times the same bug in the past few months. And encouraged us to start refactoring.